SPARKCONF.md

```diff
@@ -0,0 +1,155 @@
+##Spark Configuration
+
+oni-ml main component uses Spark and Spark SQL to analyze network events and produce a list of least probable events
+or most suspicious. 
+
+Before running oni-ml users need to configure the component. Here are our recommended settings.  
+
+### Yarn tuning
+
+oni-ml Spark application has been developed and tested on CDH [Yarn](http://spark.apache.org/docs/latest/running-on-yarn.html) 
+clusters. Careful tuning of the Yarn cluster may be necessary before analyzing large amounts of data with oni-ml.
+
+For small data sets, perhaps a couple of hundreds of gigabytes of CSV data, default _Yarn_ configurations should be enough
```

Comment:
"perhaps a couple of hundreds of gigabytes"

just give some numbers for what what we've run as a small dataset on a small cluster when testing

eg.  flow data set YYYYMMDD had size XX GB and we ran it on an M node cluster with X processers and Y gigs of RAM per node...